### PR TITLE
Portfolio: setup and mobile first project

### PR DESCRIPTION
**Pull request by adeola003**
In order to complete the project, I followed the following steps :
1. I chose template 1 on Figma and downloaded the files I would need for the project (images, icons ...);
2. I created the HTML file, built the skeleton, and filed it based on template 1 of the microverse project on Figma;
3. I created the CSS file and started styling my elements according to the template by copying the styles used for each element.
4. I added the linters for the HTML and CSS projects and fixed the detected errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  <meta 
+  name="Description"
+  content="First project creating the first two sections of my Portfolio">
+  <title>Abdel-Khafid's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="nav-container">
+        <div class="nav-logo">
+            <img class="my-lo" src="./assets/Abdel khafid SALAOU.png" alt="My_logo">
```
Comment (algerina) [inline index.html]: - [x] You did a great job however,  one of the requirements is to put the logo on the left and in a link `a` tag. This is for three reasons: convention, semantics, and user experience(UX), for further information you can check the [link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                        <a href="#Portfolio" class="link">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#About" class="link">About</a>
+                    </li>
+                    <li>
+                        <a href="#Contact" class="link">Contact</a>
+                    </li>
+                </ul>
+            </nav>
```
Comment (algerina) [inline index.html]: - [ ] It is required to have a hamburger button on the right, and inside a `nav` tag. in your project there is no hamburger menu. Kindly implement it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                    <h1>I'm Abdel-Khafid SALAOU</h1>
+                    <p>
+                        I am a Software developer! I can help you build a product,
+                        feature or website. Look through some of my works and
+                        experience! if you like what you see and have a project you
+                        need coded don't hesitate to contact me.
+                    </p>
+                </div>
+                <div class="connect">
+                    <p>LET'S CONNECT</p>
```
Comment (algerina) [inline index.html]: - [x] I recommend changing the `p`  to an `a` tag.

File: README.md [hunk 4]
```diff
@@ -1 +1,145 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (algerina) [inline README.md]: - [ ] This section should not be added as an image, kindly use the Microverse template correctly, the author's social media should be clickable links as well as the MIT license.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @adeola003  ,

Good job so far!  But still some minor issues to complete.

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- No linters errors:heavy_check_mark:



## STATUS: REQUIRED CHANGES 




 ##required changes:

- Please check the comments under the review.

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: README.md (hunk 4 above)
Comment (adeola003) [reply README.md]: @algerina  Good morning sir. thank you for your time and for helping me to get better at this. I followed your recommendations on the project and I'm about to submit a new review request.
Best regards.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  <meta 
+  name="Description"
+  content="First project creating the first two sections of my Portfolio">
+  <title>Abdel-Khafid's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="nav-container">
+        <div class="nav-logo">
+            <img class="my-lo" src="./assets/Abdel khafid SALAOU.png" alt="My_logo">
```
Comment (Lizdev-05) [inline index.html]: Great job so far 👍 . As stated by the previous reviewer, the logo should be on the left and in a link a tag. This is for three reasons: convention, semantics, and user experience(UX), for further information you can check the[ link.](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)
```
<header>
 <a href="#Portfolio" class="link"><img class="my-lo" src="./assets/Abdel khafid SALAOU.png" alt="My_logo"></a>
<nav class="topnav">
</nav>
</header>

```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                    <h1>I'm Abdel-Khafid SALAOU</h1>
+                    <p>
+                        I am a Software developer! I can help you build a product,
+                        feature or website. Look through some of my works and
+                        experience! if you like what you see and have a project you
+                        need coded don't hesitate to contact me.
+                    </p>
+                </div>
+                <div class="connect">
+                    <p>LET'S CONNECT</p>
```
Comment (Lizdev-05) [inline index.html]: Kindly change the `<p> ` tag to an `<a>` tag as suggested by the previous reviewer. It should be a link that will link to your contact page once clicked. 😞 ☺️

File: README.md [hunk 7]
```diff
@@ -1 +1,141 @@
...
+All my 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is <a href="https://github.com/adeola003/Microverse_portfolio/blob/3ca5489580c13cb372bd064c1f13825c2d17ba40/LICENSE.md">MIT licensed</a>.
```
Comment (Lizdev-05) [inline README.md]: Your MIT license is not properly linked. Kindly download the [template ](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)available on GitHub and link it like this 👇 
This project is [MIT](./LICENSE) licensed.

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @adeola003, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- Correct GitHub Flow 👍

- Good PR title/description 💯

- All linters passed 🏅


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 7 above)
Comment (adeola003) [reply README.md]: Okay, Madam @Lizdev-05. So if I understand you well, I just have to remove "licensed" from the "a" element and put it just after the closing tag. is that right?
Thank you for your time @Lizdev-05.

File: index.html (hunk 5 above)
Comment (adeola003) [reply index.html]: Thank you @Lizdev-05 it will be done.

File: index.html (hunk 6 above)
Comment (adeola003) [reply index.html]: Sorry for that I was so in a rush to complete the project and start today's one.

File: README.md (hunk 7 above)
Comment (adeola003) [reply README.md]: @Lizdev-05 I implemented all the changes I will be waiting for your confirmation to submit it as this is my last authorized review request.

**Review by iambenkis — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @adeola003 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

- [ ] Correct GitHub Flow :heavy_check_mark: 
- [ ] Good PR title/description :heavy_check_mark: 
- [ ] All linters passed :heavy_check_mark: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**